paragraph/simulator/simple_sim.cc

```diff
@@ -37,8 +37,12 @@ shim::StatusOr<std::unique_ptr<SimpleSim>> SimpleSim::Create(
 }
 
 absl::Status SimpleSim::StartSimulation(double start_time) {
```

Comment:
This should really be called Simulate(). StartSimulation() leads you to believe it is a non-blocking function call.